spec/system/create_work_draft_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Create a work draft' do
+  it 'creates a work draft' do
+    visit new_work_path
+
+    expect(page).to have_css('h1', text: 'Untitled deposit')
+
+    # Testing tabs
```

Comment:
Testing in a system test because what we really want to test is that the tabs work correctly (js).

Presumably, more will get added to this system test as we make progress.